src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -242,7 +250,6 @@ void CodeGen::genHWIntrinsic_R_R_RM(GenTreeHWIntrinsic* node, instruction ins)
                     offset = 0;
 
                     // Ensure that all the GenTreeIndir values are set to their defaults.
-                    assert(memBase->gtRegNum == REG_NA);
```

Comment:
This was found to be incorrect in the larger `emitInsBinary` refactoring, but wasn't also removed from here.